Duplicati/Server/Runner.cs

```diff
@@ -837,19 +840,19 @@ private static void UpdateMetadata(Duplicati.Server.Serialization.Interface.IBac
                     );
                 }
             }
-            else if (result.ParsedResult != Library.Interface.ParsedResultType.Success)
+            else if (result.ParsedResult != ParsedResultType.Success)
             {
-                var type = result.ParsedResult == Library.Interface.ParsedResultType.Warning
+                var type = result.ParsedResult != ParsedResultType.Warning

```

Comment:
@Jojo-1000 

Err, I know I'm bit late but I suddenly wonder about this change. Why did you replace == by != exactly (same for the 2 following changes) ? 
It seems wrong. And when running a repair from the Web UI with a warning, it displays a red window saying '0 errors'

Reply:
Yes I didn't catch that. The intention was to handle fatal the same as error, so I wanted to reverse the ternary operator. Apparently I forgot to reverse the order of the condition. But now that I look at this it would have worked in the original version anyways.